docs/content/en/docs-dev/operator-manual/piped/configuration-reference.md

```diff
@@ -24,8 +24,9 @@ spec:
 | pipedKeyFile | string | The path to the file containing the generated key string for this piped. | Yes |
 | pipedKeyData | string | Base64 encoded string of Piped key. Either pipedKeyFile or pipedKeyData must be set. | Yes |
 | apiAddress | string | The address used to connect to the control-plane's API. | Yes |
-| syncInterval | duration | How often to check whether an application should be synced. Default is `1m`. | No | <!-- TODO: Add doc for appConfigSyncInterval -->
-| git | [Git](/docs/operator-manual/piped/configuration-reference/#git) | Git configuration needed for Git commands.  | No |
+| syncInterval | duration | How often to check whether an application should be synced. Default is `1m`. | No |
+| appConfigSyncInterval | duration | How often to check whether application configuration files should be synced. Default is `5m`. | No |
+| git | [Git](/docs/operator-manual/piped/configuration-reference/#git) | Git configuration needed for Git commands. | No |
```

Comment:
```suggestion
| git | [Git](#git) | Git configuration needed for Git commands. | No |
```
nits, should refer to the tag of the same page, not jump to other version docs

Reply:
Good catch!